package.json

```diff
@@ -27,6 +27,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-icons": "^4.11.0",
+    "react-snowfall": "^2.1.0",
```

Comment:
remove this package, we dont need it probably

Reply:
Done ✅